codec_registry.src.html

```diff
@@ -0,0 +1,172 @@
+<pre class='metadata'>
+Title: WebCodecs Codec Registry
+Repository: wicg/web-codecs
+Status: CG-DRAFT
+Shortname: web-codecs-codec-registry
+Level: none
+Group: wicg
+ED: none
+Editor: Chris Cunningham, w3cid 114832, Google Inc. https://google.com/
+Editor: Paul Adenot, w3cid 62410, Mozilla https://www.mozilla.org/
+Editor: Bernard Aboba, w3cid 65611, Microsoft Corporation https://www.microsoft.com/
+
+Boilerplate: omit conformance
+
+Abstract: This registry is intended to enhance interoperability among
+    implementations and users of [[WebCodecs]]. In particular, this
+    registry provides the means to identify and avoid collisions among codec
+    strings and provides a mechanism to define codec-specific members of
+    [[WebCodecs]] codec configuration dictionaries.
+
+    This registry is not intended to include any information on whether a codec
+    format is encumbered by intellectual property claims. Implementers and
+    users are advised to seek appropriate legal counsel in this matter if they
+    intend to implement or use a specific codec format. Implementers of
+    WebCodecs are not required to support any particular codec nor registry
+    entry.
+
+    This registry is non-normative.
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/wicg/web-codecs">Git Repository.</a>
+!Participate: <a href="https://github.com/wicg/web-codecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/wicg/web-codecs/commits">https://github.com/wicg/web-codecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: web-codecs; for: WebCodecs; urlPrefix: https://wicg.github.io/web-codecs/#
+    type: attribute
+        text: AudioDecoderConfig.description; url: dom-audiodecoderconfig-description
+        text: VideoDecoderConfig.description; url: dom-videodecoderconfig-description
+        text: VideoEncoderConfig; url: video-encoder-config
+        text: AudioEncoderConfig; url: audio-encoder-config
+</pre>
+
+<pre class='biblio'>
+{
+  "WebCodecs": {
```

Comment:
The other document has "WEB-CODECS". In general, it is good practice to use the same name as the spec's shortname, and good practice to use the repo name as spec shortname, so "web-codecs". If you would like to change all names to "webcodecs", now would be a good time as we transition the repo and spec to the W3C org.

Reply:
I was more thinking of string match regardless of case rather than exact match. I do not think that there is a written recommendation about the caps style for references (for instance, I don't see any in the [Manual of Style](https://w3c.github.io/manual-of-style/#References)). A majority of specs use upper case for biblio refs. I personally prefer that too, but up to you!

To be clear, there are no written guidelines that I'm aware of to recommend that biblio refs should use the shortname of the spec, which should also match the repo name. Given that naming is notoriously hard, it just seems a good idea to be consistent.

Noted for the change of name. Let's drop the `-` during repo transition!

Reply:
Thanks. Updated all to use UPPER case.

Reply:
Thanks. Updated all to use UPPER case.